specs/src/ante_handler_v3.md

```diff
@@ -0,0 +1,25 @@
+# AnteHandler v3
```

Comment:
this file won't show up in the markdown book because it isn't listed in the summary.

For subsequent PRs: can you please try running `mdbook serve` in the root directory of the specs? Then you can visually verify that the specs match what you expect them to look like.

Reply:
Apologies. I haven't spent a lot of time looking at specs. I'll go through it properly.

Reply:
No worries! The https://github.com/celestiaorg/celestia-app/blob/main/specs/README.md should have setup steps